src/components/screens/ScoreSetView.vue

```diff
@@ -42,11 +42,39 @@
           <h3>{{ item.urn }}</h3>
         </div>
       </div>
-      <div v-if="scores" class="mave-score-set-histogram-pane">
-        <ScoreSetHistogram :scoreSet="item" :scores="scores" />
-      </div>
-      <div v-if="showHeatmap && scores" class="mave-score-set-heatmap-pane">
-        <ScoreSetHeatmap :scoreSet="item" :scores="scores" />
+      <div v-if="scores">
+        <div class="mave-score-set-variant-search">
+          <span class="p-float-label">
+              <AutoComplete
+              v-model="selectedVariant"
+              :id="$scopedId('variant-search')"
+              :suggestions="variantSearchSuggestions"
+              :option-label="variantSearchLabel"
+              dropdown
+              @complete="variantSearch"
+              selectOnFocus
+              scroll-height="100px"
+              :virtualScrollerOptions="{ itemSize: 50 }"
+              style="flex: 1; padding-right: 0.1em;"
+            >
+              <template #option="slotProps">
+                <div class="flex align-options-center">
+                  <div v-if="variantNotNullOrNA(slotProps.option.hgvs_nt)">Nucleotide variant: {{ slotProps.option.hgvs_nt }};&nbsp;</div>
+                  <div v-if="variantNotNullOrNA(slotProps.option.hgvs_splice)">Splice variant: {{ slotProps.option.hgvs_splice }};&nbsp;</div>
+                  <div v-if="variantNotNullOrNA(slotProps.option.hgvs_pro)">Protein variant: {{ slotProps.option.hgvs_pro }};&nbsp;</div>
+                  </div>
+              </template>
+            </AutoComplete>
+            <label :for="$scopedId('variant-search')">Search for a variant defined by this score set</label>
+            <Button icon="pi pi-times" severity="danger" aria-label="Clear" @click="selectedVariant = null" />
+          </span>
+        </div>
+        <div class="mave-score-set-histogram-pane">
+          <ScoreSetHistogram :scoreSet="item" :scores="scores" :externalSelection="typeof selectedVariant === 'object' ? selectedVariant : null" />
```

Comment:
having these `typeof`s here is a bit confusing. could this be a computed property?